### PR TITLE
[Debugger] Various fixes to improve a11y

### PR DESCRIPTION
**Pull request by jstedfast**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1027799/
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1027825/
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1022231/

### CODE REVIEW

**Comment by jstedfast** [comment]
@monojenkins backport release-8.4

**Comment by jstedfast** [comment]
@monojenkins backport release-8.4

**Comment by jstedfast** [comment]
Updated screenshot of the "Refresh" icon:

refresh-icon-contrast

**Comment by jstedfast** [comment]
@monojenkins backport release-8.4

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs [hunk 1]
```diff
@@ -35,16 +35,13 @@ namespace MonoDevelop.Debugger
...
-		static readonly NSImage selectedUnpinnedImage = GetImage ("md-pin-up", Gtk.IconSize.Menu, true);
-		static readonly NSImage selectedPinnedImage = GetImage ("md-pin-down", Gtk.IconSize.Menu, true);
-		static readonly NSImage unpinnedImage = GetImage ("md-pin-up", Gtk.IconSize.Menu);
-		static readonly NSImage pinnedImage = GetImage ("md-pin-down", Gtk.IconSize.Menu);
-
-		static readonly NSImage liveUpdateOnImage = GetImage ("md-live", Gtk.IconSize.Menu);
-		static readonly NSImage liveUpdateOffImage = GetImage ("md-live", Gtk.IconSize.Menu, 0.5);
-		static readonly NSImage none = GetImage ("md-empty", Gtk.IconSize.Menu);
+		static readonly NSImage EmptyImage = GetImage ("md-empty", Gtk.IconSize.Menu);
+		static readonly string [] ActiveSelectedStyle = { "active", "sel" };
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs]: Should these be constants defined somewhere in MD.Ide? cc @sevoku

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs [hunk 2]
```diff
@@ -103,24 +100,28 @@ protected override void UpdateContents ()
...
 
 			if (TreeView.PinnedWatch != null && Node.Parent == TreeView.Controller.Root) {
-				PinButton.Image = selected ? selectedPinnedImage : pinnedImage;
+				PinButton.Image = GetImage ("md-pin-down", Gtk.IconSize.Menu, selected);
 				pinned = true;
 			} else {
 				// When a11y is in use, always show an icon.
 				if (IdeServices.DesktopService.AccessibilityInUse)
-					PinButton.Image = selected ? selectedUnpinnedImage : unpinnedImage;
+					PinButton.Image = GetImage ("md-pin-up", Gtk.IconSize.Menu, selected);
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs]: We have constants for md-pin-down and md-pin-up: http://source.monodevelop.com/#MonoDevelop.Ide/MonoDevelop.Ide.Gui/StockIcons.cs,2e483ee18a808cd2,references

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs (hunk 1 above)
Comment (sevoku) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs]: We should add that for styles, but that's out of scope here for now. But we have constants for all the icons used [here](http://source.monodevelop.com/#MonoDevelop.Ide/MonoDevelop.Ide.Gui/StockIcons.cs,54).

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs (hunk 2 above)
Comment (jstedfast) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs]: Updated to use the constants.

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs (hunk 1 above)
Comment (jstedfast) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ObjectValue/Mac/MacDebuggerObjectPinView.cs]: I updated the code yesterday to use Stock.Empty.

**Comment by jstedfast** [comment]
@monojenkins backport release-8.4

**Comment by sevoku** [comment]
@monojenkins backport release-8.4-a11y

**Pull request merged**